app/views/posts/show.html.erb

```diff
@@ -17,4 +20,9 @@
                 <span><%= comment.text %></span>
             <div>  
     <% end %>
- <div>
\ No newline at end of file
+ <div>
+
+<div class='add_comment_wrapper'>
+ <%= render "comments/form" %>
+</div>
+
```

Comment:
- [ ] Rather than rendering the form for creating a comment on the posts show page, the form should be in the `views/comments/new.html.erb` file which accepts input for creating comments.

Reply:
The same has been applied here as they have the same connection. Thank you.